recipes/cmake-nocurl/meta.yaml

```
@@ -0,0 +1,52 @@
+{% set version = "3.16.4" %}
+
+package:
+  name: cmake-no-system
+  version: {{ version }}
+
+source:
+  url: https://gitlab.kitware.com/cmake/cmake/-/archive/v{{ version }}/cmake-v{{ version }}.tar.bz2
+  sha256: 40e0dec6dc9e36820e001b8425aa4328a0b42f1915b14d68aee116e25c3d34df
+  patches:
+    - patches/3.16.2/0001-find_-Add-debug-logging-infrastructure.patch
+    - patches/3.16.2/0002-find_-Use-debug-logging-infrastructure.patch
+    - patches/3.16.2/0003-Add-more-debug-logging-to-cmFindCommon.patch
+
+build:
+  number: 0
+  skip: True  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make
+    - patch
+
+test:
+  commands:
+    - cmake --version
+    - ctest --version
+    - cpack --version
+
+about:
+  home: http://www.cmake.org/
+  license: BSD 3-clause
+  license_family: BSD
+  license_file:
+    - Copyright.txt
+    - Utilities/cmbzip2/LICENSE
+    - Utilities/cmcurl/COPYING
+    - Utilities/cmexpat/COPYING
+    - Utilities/cmjsoncpp/LICENSE
+    - Utilities/cmlibarchive/COPYING
+    - Utilities/cmliblzma/COPYING
+    - Utilities/cmlibrhash/COPYING
+    - Utilities/cmlibuv/LICENSE
+    - Utilities/cmzlib/Copyright.txt
+    - Utilities/cmzstd/LICENSE
+  summary: CMake built without system libraries for use when building CMake dependencies.
+
+extra:
+  recipe-maintainers:
+    - jjhelmus
```

```suggestion
    - jjhelmus
    - beckermr
```

count me in to help out if I can!